src/ucs/config/parser.h

```diff
@@ -322,7 +322,7 @@ void ucs_config_help_generic(char *buf, size_t max, const void *arg);
 #define UCS_CONFIG_TYPE_STRING_ARRAY \
     UCS_CONFIG_TYPE_ARRAY(string)
 
-UCS_CONFIG_DECLARE_ARRAY(string);
+UCS_CONFIG_DECLARE_ARRAY(string)
```

Comment:
may be remove ```;``` from macro declaration?

Reply:
@hoopoepg 
Makes no difference to me, but that would mean every mention of `UCS_CONFIG_DECLARE_ARRAY` will have to be followed by a semicolon (we need one somewhere), which is a bit different from UCX's coding style IIRC... 
I assume there's no use-case for this macro without a semicolon at it's end, right?

Reply:
i think it's ok to just remove from here now